framework/source/class/qx/bom/Cookie.js

```diff
@@ -62,7 +62,7 @@ qx.Bootstrap.define("qx.bom.Cookie",
         end = document.cookie.length;
       }
 
-      return decodeURI(document.cookie.substring(len, end));
+      return unescape(document.cookie.substring(len, end));
     },
```

Comment:
Yes. I'll try to do it right in my next try. Squash is absolutely something new for me. :sweat_smile: